### PR TITLE
splitStrong/Choice with Semigroupoid instead of Category constrain

### PR DESCRIPTION
**Pull request by erykciepiela**
**Description of the change**

#47 

---

**Checklist:**

- [ x] Added the change to the changelog's "Unreleased" section with a reference to this PR (e.g. "- Made a change (#0000)")
- [ x] Linked any existing issues or proposals that this pull request should close
- [ ] Updated or added relevant documentation
- [ ] Added a test for the contribution (if applicable)

### CODE REVIEW

**Review by garyb — APPROVED** [review]

**Comment by erykciepiela** [comment]
@garyb thanks! what needs to be done for it to merge and publish?

**Comment by garyb** [comment]
We're [supposed to](https://github.com/purescript/governance?tab=readme-ov-file#pull-requests) get two approvals from core team members/collaborators before merging/releasing changes to these libraries, so just waiting on that. :wink:

@MonoidMusician or @rhendric maybe?

**Review by MonoidMusician — APPROVED** [review]

**Pull request merged**
